backend/final_e2e_test_with_scoring.py

```
@@ -69,19 +69,25 @@ async def run_complete_test():
-            f"{backend_url}/api/generate-fallback-guide/{trip_id}",
-            timeout=15
+            f"{backend_url}/api/generate-guide/{trip_id}",
+            timeout=30
         )
         
         if response.status_code == 200:
             result = response.json()
-            guide = result.get("guide")
-            guide_type = "fallback"
-            print("✅ Fallback guide generated successfully!")
+            guide_response = requests.get(f"{backend_url}/api/enhanced-guide/{trip_id}")
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)